packages/ComponentLibrary/src/components/NotificationsModal/styles.ts

```diff
@@ -0,0 +1,94 @@
+import { CSSProperties } from 'react';
+
+const NEUTRAL_10 = '#F5F6FA';
```

Comment:
These styles are already defined in colors.ts file, it would be better to use them from there or add them there and use them here